etc/calc_release_version.py

```diff
@@ -196,16 +196,19 @@ def process_and_sort_tags(tags):
     for tag in raw_tags:
```

Comment:
Update comment above to replace `LooseVersion comparison` with `Version comparison`

Reply:
I pushed a change based on mongodb/mongo-c-driver@c35519cc5fb0

Reply:
And another commit to fix the comment.